src/background/fxa.js

```diff
@@ -502,6 +502,8 @@ export class FxAUtils extends Component {
     url.searchParams.set("uid", profileData.uid);
     // eslint-disable-next-line verify-await/check
     url.searchParams.set("email", profileData.email);
+    // eslint-disable-next-line verify-await/check
+    url.searchParams.set("entrypoint", "secure-proxy");
```

Comment:
Since there will eventually be secure-proxy on Android too, can you make this `secure-proxy-desktop-settings`? Having the entrypoints as specific as possible really make bucketing a lot easier.